lib/package-manager.coffee

```diff
@@ -309,7 +309,7 @@ class PackageManager
     nameWithVersion = if version? then "#{name}@#{version}" else name
 
     @unload(name)
-    args = ['install', nameWithVersion, '--json']
+    args = ['install', nameWithVersion, '--production', '--json']
```

Comment:
^ the only actual change. The rest are changes to the tests to account for the new apm install flag